common-libs/vc-common/package.json

```diff
@@ -16,14 +16,13 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "start": "tsdx watch --tsconfig ./tsconfig.build.json",
```

Comment:
Why did you remove start?

Reply:
It is a library, what is to start here? The original scripts were auto-generated by tsdx, they didn't make any sense then and don't make any sense now that tsdx is removed